inflammation/models.py

```diff
@@ -32,3 +32,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
```

Comment:
This function reimplements a calculation for standard deviation, for which other packages already have functions for. For example numpy's np.std() could be used instead.